### DISCUSSION TITLE
Add Fingerprint for Ford Explorer?

### DISCUSSION THREAD

**Discussion by richardkaplan** [community/community#89797] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Greetings - I would like to try Frogpilot on my 2023 Ford Explorer.  Currently I am using Openpilot on my own custom fork so my car's fingerprint is recognized; there seems to be a bit of a delay for OpenPilot to commit pull requests for fingerprints.

Is there any way I can use  Frogpilot until my fingerprint pull request has been added to OpenPilot?

https://github.com/commaai/openpilot/pull/31062

**Comment by queenofcorgis** [comment, marked as answer]
Hi @richardkaplan ,

Thanks for being a part of the GitHub Community, we're glad you're here! The GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic as the Community nor GitHub provides support or answer questions on projects hosted on GitHub.
